src/java/picard/analysis/CollectAlignmentSummaryMetrics.java

```diff
@@ -78,10 +78,11 @@
         programGroup = Metrics.class
 )
 public class CollectAlignmentSummaryMetrics extends SinglePassSamProgram {
-    static final String USAGE_SUMMARY = "Produce a summary of alignment metrics from a SAM or BAM file";
-    static final String USAGE_DETAILS = "Using read outputs from high throughput sequencing (HTS) technologies, this tool provides " +
-            "metrics regarding the quality of read alignments to a reference sequence, as well as the proportion of the reads " +
-            "that passed machine signal-to-noise threshold quality filters (Illumina)."+
+    static final String USAGE_SUMMARY = "Produces a summary of alignment metrics from a SAM or BAM file";
+    static final String USAGE_DETAILS = "This tool takes an aligned SAM/BAM file, and produces metrics detailing the quality of the read alignments" +
+            " as well as the machine-generated base quality scores within these reads.  Additional information about Illumina's base quality " +
+            "scoring and the filters applied to these scores can be found at the following " +
+            "<a href=\"http://gatkforums.broadinstitute.org/gatk/discussion/6329/pf-reads-illumina-chastity-filter\"<b>link</b></a>."+
```

Comment:
I just figured it out, you're totally fine -- you're looking at it in the "commits" tab, and there indeed there's changes to the .iml file in your second commit -- but they're the on-purpose changes we made (with `checkout --`) that *undo* the badness of your first commit. Overall they cancel each other out, which you can see by looking at the "files changed" at the PR-level (above commit-level). Once you rebase (when review is done) it'll all be a-okay.